### ISSUE TITLE
[CoincSuite] Fix failing test (Trac #1539)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1531]
Figure out where the comparison between either two I3Particles, I3Positions, or I3Directions is causing the result to change and renable recursionTest.py.

Migrated from https://code.icecube.wisc.edu/ticket/1539
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Figure out where the comparison between either two I3Particles, I3Positions, or I3Directions is causing the result to change and renable recursionTest.py.", 
    "reporter": "olivas", 
    "cc": "david.schultz", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[CoincSuite] Fix failing test", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-02-04T16:19:05", 
    "milestone": "", 
    "owner": "mzoll", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160204T16:54:25: olivas commented:

> For ho-hos I replaced CompareFloatingPoint::IsNan with std::isnan and that test still fails.

**Comment by icecube-trac** [comment]
Trac update at 20160426T12:43:17: mzoll commented:

> Well I can regenerate the file to compare to and define this as the new default state. It has just be be put to the I3TESTDATA and things should be fixed.
> 
> The origin of the change, caused by something underneath my project, will remain a mystery I guess

**Comment by icecube-trac** [comment]
Trac update at 20160428T13:22:26: 
* mzoll commented:

> In changeset 145131:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145131"
> the error is burried in a comparision between I3Positions, where a exceptional state was raised becaused of double precision hopping over a previous check; fixes #1531
> ```


* mzoll changed status from "new" to "closed"
* mzoll changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
